packages/server-wallet/src/protocols/process-ledger-queue.ts

```diff
@@ -5,7 +5,6 @@ import {
   checkThat,
```

Comment:
I don't see any unit tests for this file. Because of that, it's difficult to understand what's going on, since this file contains a large amount of complex code.